tools/docker-compose/supervisor.conf

```diff
@@ -91,7 +101,7 @@ stdout_events_enabled = true
 stderr_events_enabled = true
 
 [group:tower-processes]
-programs=awx-dispatcher,awx-receiver,awx-uwsgi,awx-daphne,awx-nginx,awx-wsrelay,awx-rsyslogd,awx-heartbeet
+programs=awx-dispatcher,awx-receiver,awx-uwsgi,awx-daphne,awx-nginx,awx-wsrelay,awx-rsyslogd,awx-heartbeet,awx-cache-clear
```

Comment:
I don't think `awx-clear-cache` necessarily needs to be added here, because this includes the dispatcher service which will clear the cache when needed. This was needed for testing, so that might be why it's here.

Reply:
Removed. Related to the comment above https://github.com/ansible/awx/pull/13438#discussion_r1093770801 if heartbeet is here does it also need to be in supervisor.conf.j2?

Reply:
Yes, heartbeet needs to run in the web container.